cypress.json

```diff
@@ -1,5 +1,5 @@
 {
-  "baseUrl": "http://test_site:8000/",
+  "baseUrl": "http://site:8000/",
```

Comment:
why is this being changed?

Reply:
Was trying to change it on my local repo, guess I messed it up here